python/ql/test/library-tests/frameworks/FileSystemAccess/Query.ql

```diff
@@ -0,0 +1,20 @@
+import python
+import semmle.python.dataflow.new.DataFlow
+import semmle.python.Concepts
+import TestUtilities.InlineExpectationsTest
+private import semmle.python.dataflow.new.internal.PrintNode
+
+module FileSystemAccessTest implements TestSig {
+  string getARelevantTag() { result = "getAPathArgument" }
+
+  predicate hasActualResult(Location location, string element, string tag, string value) {
+    exists(location.getFile().getRelativePath()) and
+    exists(FileSystemAccess a, DataFlow::Node path |
+      path = a.getAPathArgument() and
+      location = a.getLocation() and
+      element = path.toString() and
+      value = prettyNodeForInlineTest(path) and
+      tag = "getAPathArgument"
+    )
+  }
+}
```

Comment:
I think you can delete this entire file, since the tests are already available in ConceptsTests: https://github.com/github/codeql/blob/991d5cc54b27338cfcd9e1cd685c425e8e7aa11a/python/ql/test/experimental/meta/ConceptsTest.qll#L375-L388